sdcm/results_analyze/__init__.py

```diff
@@ -222,15 +222,22 @@ def check_regression(self, test_id, data, is_gce=False):  # pylint: disable=too-
             build_id = ''
 
         last_events, events_summary = self.get_events(event_severity=[
-            Severity.CRITICAL.name, Severity.ERROR.name, Severity.WARNING.name])
-        reactor_stall_events, reactor_stall_events_summary = self.get_debug_events()
+            Severity.CRITICAL.name, Severity.ERROR.name])
+        debug_events, _ = self.get_debug_events()
+        reactor_stall_events = [stall for stall in debug_events[Severity.DEBUG.name] if 'type=REACTOR_STALLED' in stall]
+        reactor_stall_events_summary = {Severity.DEBUG.name: len(reactor_stall_events)}
+        kernel_callstack_events = \
+            [stall for stall in debug_events[Severity.DEBUG.name] if 'type=KERNEL_CALLSTACK' in stall]
+        kernel_callstack_events_summary = {Severity.DEBUG.name: len(kernel_callstack_events)}
```

Comment:
WDYT, will it be better if it will be functions : get_reactor_stall_events, get_kernel_call_stack_events?

Reply:
changed